### PR TITLE
feat: Inject document service URL into Tasklist forms

### LINKED ISSUE #26221: Integrate `form-js` document preview with Tasklist
### Description

On [`form-js@1.13.0`](https://github.com/bpmn-io/form-js/blob/develop/packages/form-js/CHANGELOG.md#1130) we released the new document .
Because of that we need to plug our document service into `form-js`. For better user experience we will inject the context variable `defaultDocumentsEndpointKey` when rendering the forms from the Tasklist frontend which will have the self-managed or saas cluster URL.
If Zeebe already has the `defaultDocumentsEndpointKey` variable available then the Zeebe variable will take precedence over the one injected by the Tasklist frontend

### TODO

- Inject the `defaultDocumentsEndpointKey` into the `form-js` context so `form-js` can pull our document service

### Additional Information

---------------------------------------------------------------------------------------------

### Acceptance Criteria

- [ ] Checked if the specific data needs to be provided for the QA process

### Definition of Ready - Checklist

- [ ] The issue has a meaningful title, description and testable acceptance criteria

### PR DESCRIPTION
**Pull request by vsgoulart**
## Description

This [Renovate PR](https://github.com/camunda/camunda/pull/26166) updated `form-js` to the latest version which introduces the new document preview component.
For this  component to work we need to inject the document service URL into the `form-js` context so the form can pull the documents

## Checklist

- [ ] for CI changes:
  - [ ] structural/foundational changes signed off by [CI DRI](https://github.com/cmur2)
  - [ ] [ci.yml](https://github.com/camunda/camunda/blob/main/.github/workflows/ci.yml) modifications comply with ["Unified CI" requirements](https://github.com/camunda/camunda/wiki/CI-&-Automation#workflow-inclusion-criteria)

## Related issues

closes #26221

### CODE REVIEW

File: tasklist/common/src/main/java/io/camunda/tasklist/property/SecurityProperties.java [hunk 1]
```diff
@@ -9,7 +9,7 @@
...
   public static final String CONTENT_SECURITY_POLICY =
-      "default-src 'self';connect-src 'self' https: *.mixpanel.com cloudflareinsights.com *.appcues.net wss://api.appcues.net;script-src 'self' https: *.chargebee.com *.mixpanel.com ajax.cloudflare.com static.cloudflareinsights.com;style-src 'self' https: 'unsafe-inline' *.googleapis.com *.chargebee.com;img-src * data: ;font-src 'self' data: https://fonts.gstatic.com https://fonts.camunda.io;frame-ancestors;frame-src 'self' https: *.chargebee.com ;child-src;worker-src 'self' blob:;base-uri 'self';form-action 'self';object-src 'none';script-src-attr 'none';";
+      "default-src 'self';connect-src 'self' https: *.mixpanel.com cloudflareinsights.com *.appcues.net wss://api.appcues.net;script-src 'self' https: *.chargebee.com *.mixpanel.com ajax.cloudflare.com static.cloudflareinsights.com;style-src 'self' https: 'unsafe-inline' *.googleapis.com *.chargebee.com;img-src * data: ;font-src 'self' data: https://fonts.gstatic.com https://fonts.camunda.io;frame-ancestors 'self';frame-src 'self' https: *.chargebee.com ;child-src;worker-src 'self' blob:;base-uri 'self';form-action 'self';object-src 'self';script-src-attr 'none';";
```
Comment (vsgoulart) [inline tasklist/common/src/main/java/io/camunda/tasklist/property/SecurityProperties.java:12]: @marcosgvieira In the end I had to change the CSP config

Comment (marcosgvieira) [reply tasklist/common/src/main/java/io/camunda/tasklist/property/SecurityProperties.java:12]: @vsgoulart this is required to load pdfs, correct?

Comment (vsgoulart) [reply tasklist/common/src/main/java/io/camunda/tasklist/property/SecurityProperties.java:12]: Yes, correct

**Review by marcosgvieira — APPROVED** [review]
It's ok for now the CSP.

**Review by pedesen — APPROVED** [review]
LGTM!

**Pull request merged**